templates/sios-datakeeper-master.template

```diff
@@ -898,15 +813,7 @@
                         "Ref": "QSS3BucketName"
                     },
                     "QSS3KeyPrefix": {
-                        "Fn::Join": [
-                          	"/",
-                           [
-                             	{
-                                	"Ref": "QSS3KeyPrefix"
-                             	},
-                              	"submodules/quickstart-microsoft-rdgateway"
-                           ]
-                        ]
+                        "Fn::Sub": "${QSS3KeyPrefix}submodules/quickstart-microsoft-rdgateway"
                     },
                     "RDGW1InstanceType": {
                         "Ref": "RDGW1InstanceType"
```

Comment:
Remember to also remove those parameters from this master template and also from the groups and labels